src/store/index.js

```diff
@@ -0,0 +1,22 @@
+import Vue from "vue";
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    stateMovies: [],
```

Comment:
no need to call it `stateMovies` because it is already in the `state` object so it would actually be `state.stateMovies`, which is a bit of a repetition, isn't it?